frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -81,12 +87,16 @@ const getColumns = () => {
 const getRows = (nodes: NodeKind[]) => {
   return nodes.map((node) => {
     const roles = getNodeRoles(node).sort();
+    const cpuCores = _.get(node.status, 'capacity.cpu');
+    const memory = _.get(node.status, 'allocatable.memory');
```

Comment:
I'd use a more precise name (and please add types since `_.get` will return `any`)

```suggestion
    const allocatableMemory: string = _.get(node.status, 'allocatable.memory');
```